File: sdkwork-ai-sdk-java/src/main/java/com/sdkwork/ai/model/ThreadResponse.java
package com.sdkwork.ai.model;

public class ThreadResponse {
    private String id;
    private String object;
    private Map<String, String> metadata;
    private Integer createdAt;
    private Map<String, Object> toolResources;

    public String getId() {
        return this.id;
    }
    
    public void setId(String id) {
        this.id = id;
    }

    public String getObject() {
        return this.object;
    }
    
    public void setObject(String object) {
        this.object = object;
    }

    public Map<String, String> getMetadata() {
        return this.metadata;
    }
    
    public void setMetadata(Map<String, String> metadata) {
        this.metadata = metadata;
    }

    public Integer getCreatedAt() {
        return this.createdAt;
    }
    
    public void setCreatedAt(Integer createdAt) {
        this.createdAt = createdAt;
    }

    public Map<String, Object> getToolResources() {
        return this.toolResources;
    }
    
    public void setToolResources(Map<String, Object> toolResources) {
        this.toolResources = toolResources;
    }
}

File: sdkwork-ai-sdk-typescript/src/types/thread-response.ts
/** 线程响应 */
export interface ThreadResponse {
  /** 线程ID */
  id?: string;
  /** 对象类型 */
  object?: string;
  /** 元数据 */
  metadata?: Record<string, string>;
  /** 创建时间(Unix时间戳) */
  created_at?: number;
  /** 工具资源 */
  tool_resources?: Record<string, unknown>;
}
